### PR TITLE
Fix softmax bug

### PR DESCRIPTION
**Pull request by QRabbani**
The maximum and summation axis was not provided leading to an incorrect output for the softmax function. Additionally, the correct output from the graph should be "raw_logits:0" (the pre-softmax output) not "logits:0" (the post-softmax output). This way, the softmax can continue to be applied manually as in the original code.

### CODE REVIEW

**Comment by QRabbani** [comment]
Thanks, by the way, for creating this script in the first place and for updating it to work with the latest model!
